frontend/components/LastUpdatedText/LastUpdatedText.tsx

```diff
@@ -6,14 +6,27 @@ import TooltipWrapper from "components/TooltipWrapper";
 
 const baseClass = "component__last-updated-text";
 
-interface ILastUpdatedTextProps {
+interface ILastUpdatedTextBase {
   lastUpdatedAt?: string;
```

Comment:
Is it possible to make this required rather than `?`

Reply:
Good question! Digging through the code, it looks like there's only one place (`CurrentVersionSection`) where this component can be rendered with the prop as undefined. I'm going to leave it in as it feels safer since it'll return the empty state defined in the component.